host/class/uac/usb_host_uac/uac_host.c

```diff
@@ -321,6 +321,26 @@ static uac_device_t *get_uac_device_by_addr(uint8_t addr)
     return NULL;
 }
 
+/**
+ * @brief Return UAC Device from USB device VID and PID
+ * @param[in] vid  USB device VID
+ * @param[in] pid  USB device PID
+ * @return uac_device_t Pointer to UAC Device
+ */
+static uac_device_t *get_uac_device_by_vid_pid(uint16_t vid, uint16_t pid)
+{
+    uac_iface_t *interface = NULL;
```

Comment:
The VID PID info is stored in interface object (Because we treat interface as a logic device in this driver), So we go through the interfaces to find the handle of the corresponding (parent) device